src/components/portal/portal.tsx

```diff
@@ -77,11 +79,14 @@ export class Portal {
     @Prop()
     public parent: HTMLElement = document.body;
 
+    /**
+     *
+     */
     @Prop()
     public inheritParentWidth = false;
```

Comment:
Note to self: add a description here (Jens and I added this in the date-picker branch).